src/main/java/uk/gov/companieshouse/api/strikeoffobjections/model/entity/Objection.java

```diff
@@ -21,6 +21,8 @@
         private ObjectionStatus status;
         private Long actionCode;
         private String httpRequestId;
+        private String fullName;
```

Comment:
In terms of Mongo, should 'fullName' and 'shareIdentity' maybe live in the 'CreatedBy' bucket/object?

Reply:
That sounds better, have the fullName and shareIdentity in CreatedBy as it is user identity related

Reply:
Yep. It all sounds related.

Reply:
Put the new fields into createdBy deleted the user details class